Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM python:3-slim-buster as base
+
+ENV user_name="app-user"
```

Comment:
This looks like it's going to set a non-root user, but I can't see anywhere that actually follows through on that!

Reply:
yes, it's leftover stuff from the ansible exercise.  I'll take a look

Reply:
removed in b3fc905